oas-generator-spi/oas-generator-asciidoctor/src/main/resources/freemarker/openapi.ftlh

```diff
@@ -10,31 +10,20 @@
 <#--
 
 -->
-<#if standalone>
 <@section title=openapi.info.title level=1/>
 <#if openapi.info.contact??>
-${openapi.info.contact.name} <${openapi.info.contact.email}>
-</#if>
-:last-update-label: Generated by `OAS Generator` based on OpenAPI Specification (v${openapi.openapi}) on
-:revnumber: ${openapi.info.version}
-:icons: ${attributeIcons}
-:toc: left
-:toclevels: 3
-
-<#if openapi.info.description??>
-${openapi.info.description}
-</#if>
-<#else>
-<@section title="OpenAPI Documentation" level=1/>
-
-<#if openapi.info.contact??>
-${openapi.info.contact.name} (${openapi.info.contact.email})
-</#if>
-
-<#if openapi.info.description??>
-
-${openapi.info.description}
+    ${openapi.info.contact.name} <${openapi.info.contact.email}>
```

Comment:
All whitespaces on the start of each line should be removed. Whitespace characters are still present in the generated file and the asciidoc format is not valid.

Reply:
Missed that during testing. Resolved.